EIPS/eip-X.md

```diff
@@ -0,0 +1,201 @@
+
+_`PROPORTIONAL_SLASHING_MULTIPLIER` is one-third of its final value_. This results in a lower accountable safety margin in the event of an attack. If any validators are slashed in early months of eth2, it is far more likely to be the result of user mismanagement of keys and/or issues with client software than an organized attack.
+
+_`MIN_SLASHING_PENALTY_QUOTIENT`_ is four times its final value_. This results in a lower guaranteed minimum penalty for a slashable offense and thus reduces the baseline punitive incentive to keep your system secure. As with `PROPORTIONAL_SLASHING_MULTIPLIER`, slashings during early months of eth2 are far more likely to be the result of user mismanagement or issues with client software than an organized attack.
+
+___A hard fork of the eth2 beacon chain is planned for 5 months after genesis. This hard fork will adjust `INACTIVITY_PENALTY_QUOTIENT`, `PROPORTIONAL_SLASHING_MULTIPLIER`, and `MIN_SLASHING_PENALTY_QUOTIENT` to their final, more secure values.___
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+Ethereum blocks are bursting at the seams due to increasingly high demand for decentralized applications. Ever since the first serious spikes in adoption in 2017 (cryptokitties), the Ethereum community has consistently and vocally demanded scaling solutions.
+
+Since day 0 of Ethereum, the investigation and expectation in scaling solutions is two-fold -- scale from both Layer 1 upgrades as well as Layer 2 adoption. 
+
+For details regarding specification design rationale and decisions, see the following:
+* [Serenity Design Rationale](https://notes.ethereum.org/@vbuterin/rkhCgQteN)
+* [Ben Edgington's Eth2 Annotated Spec](https://benjaminion.xyz/eth2-annotated-spec/)
+* [Vitalik Buterin's Eth2 Annotated Spec](https://github.com/ethereum/annotated-spec/blob/master/phase0/beacon-chain.md)
+
+## Backwards Compatibility
+<!--All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
+This EIP does not introduce any layer 1 changes to Ethereum mainnet and is thus entirely forwards/backwards compatible.
```

Comment:
Is modifying the issuance schedule of ether considered an L1 change?

Reply:
What I am trying to get across is that this is an extension of the protocol rather than a modification of the existing set of rules (at this phase). Open to clearer ways to get this across.

Reply:
What I am trying to get across is that this is an extension of the protocol rather than a modification of the existing set of rules (at this phase). Open to clearer ways to get this across.

Reply:
```suggestion
Although this EIP does not propose any changes to the current Ethereum mainnet, this EIP lays the groundwork for future incompatibilities . . .
```
Rereading, it seems okay given the next paragraph. It might be slightly clearer to put less emphasis on the fact that this particular EIP is backwards compatible.